README.md

```diff
@@ -139,7 +139,7 @@ compromise.
 Not in the short term, no.
 
 This project is still experimental and unpopular. While it is, in a way,
-a [provider] 2.0, its worth has yet to be proven.
+a [provider] 2.0, it's worth has yet to be proven.
```

Comment:
Original spelling was correct. `its worth has yet to be proven` because it is possessive, as in `my worth` or `your worth`. `It is worth has yet to be proven` does not make much sense.

Reply:
Fixed by #5 , please disregard